packages/db/src/models/system/pairwise-associations-occurrence.ts

```diff
@@ -36,6 +36,12 @@ export default class PAOccurrence extends BaseModel<
     type: DataType.INTEGER,
   })
   declare occurrences: number;
+
+  @Column({
+    allowNull: true,
+    type: DataType.INTEGER,
+  })
+  declare multiplier: number | null;
```

Comment:
wrap with `CreationOptional` -> CreationOptional<number | null>, it won't be optional during creation